libs/vue/README.md

```diff
@@ -51,6 +51,19 @@ nx g @nx-plus/vue:app my-app
 nx serve my-app
 ```
 
+### Generate Your Library
+
+```
+nx g @nx-plus/vue:lib my-lib
+```
+
+### Build Your Library
+
+```
+# Only applicable to libraries generated with the --publishable flag
+nx build my-lib
+```
+
```

Comment:
Remove this from getting started. The getting started section is our it's as easy as 1, 2, 3. We want to keep this as simple as possible. That's why I removed how to build an app from the section.